lib/proxy.js

```diff
@@ -27,12 +27,12 @@ const proxyLog = util.loggers.proxyLogger;
 
 function convertOptions(request, realHost, realPort, urlPrefix) {
   var options = {};
-  proxyLog.debug("request host " + request.headers.host);
+  proxyLog.debug("ZWED0166I", request.headers.host); //proxyLog.debug("request host " + request.headers.host);
```

Comment:
The json has an error
 "ZWED0166I":"request host %",
Meant %s?

Reply:
Addressed.